qubesmanager/settings.py

```diff
@@ -758,6 +763,16 @@ def __apply_advanced_tab__(self):
         except Exception as ex:  # pylint: disable=broad-except
             msg.append(str(ex))
 
+        if getattr(self.vm, "template_for_dispvms", False) != \
+                self.dvm_template_checkbox.isChecked():
+            try:
+                self.vm.template_for_dispvms = \
+                    self.dvm_template_checkbox.isChecked()
+                self.vm.features["appmenus-dispvm"] = \
+                    1 if self.dvm_template_checkbox.isChecked() else ""
```

Comment:
In case of disabling this setting, I'd remove `appmenus-dispvm` feature, instead of setting it to false.

BTW setting features to `bool` values is supported - they will be converted to appropriate values (`1`, empty) preserving true/false semantics.